packages/ui/src/components/GalleryMedia/GalleryMedia.tsx

```diff
@@ -74,3 +70,20 @@ export function GalleryMedia({ className, gallery, locale }: Props) {
         </Link>
     );
 }
+
+export namespace GalleryMedia {
+    export interface Intl {
+        ['videos.title']: string;
+        ['images.title']: string;
+    }
+
+    export interface Props {
+        galleryHref: `/${string}`;
+        name: NewsroomGallery['name'];
+        className?: string;
+        imagesNumber: NewsroomGallery['images_number'];
+        videosNumber: NewsroomGallery['videos_number'];
+        cover?: UploadedImage | null;
```

Comment:
Let's group gallery-related props into a `DisplayedGallery` object:

```ts
export interface DisplayedGallery {
   name: NewsroomGallery['name'];
   images: NewsoromGallery['images_number'];
   videos: NewsroomGallery['videos_number'];
   cover?: UploadedImage | null;
   href: string;
}
```

and use it in the `Props` declaration:

```ts
export interface Props  {
   className?: string;
   gallery: DisplayedGallery;
}
```